homeassistant/components/overkiz/const.py

```diff
@@ -61,6 +63,7 @@
     UIWidget.RTD_INDOOR_SIREN: Platform.SWITCH,  # widgetName, uiClass is Siren (not supported)
     UIWidget.RTD_OUTDOOR_SIREN: Platform.SWITCH,  # widgetName, uiClass is Siren (not supported)
     UIWidget.RTS_GENERIC: Platform.COVER,  # widgetName, uiClass is Generic (not supported)
+    UIWidget.SIREN_STATUS: None,  # widgetName, uiClass is Siren (siren)
```

Comment:
Why add this widget, if the associated platform is None? 

https://github.com/iMicknl/home-assistant-core/blob/overkiz/add_siren_platform/homeassistant/components/overkiz/__init__.py#L109

Having None, or a missing key will result in the same case.